EAS build fails at `Install dependencies` phase.

Here are the logs from EAS extracted from the `Install dependencies` step:

```
Running "npm ci" in /home/expo/workingdir/build directory

npm warn ERESOLVE overriding peer dependency

npm warn While resolving: @react-native/virtualized-lists@0.79.2

npm warn Found: @types/react@18.3.23

npm warn node_modules/@types/react

npm warn   dev @types/react@"~18.3.12" from the root project

npm warn

npm warn Could not resolve dependency:

npm warn peerOptional @types/react@"^19.0.0" from @react-native/virtualized-lists@0.79.2

npm warn node_modules/@react-native/virtualized-lists

npm warn   @react-native/virtualized-lists@"0.79.2" from react-native@0.79.2

npm warn   node_modules/react-native

npm warn

npm warn Conflicting peer dependency: @types/react@19.1.12

npm warn node_modules/@types/react

npm warn   peerOptional @types/react@"^19.0.0" from @react-native/virtualized-lists@0.79.2

npm warn   node_modules/@react-native/virtualized-lists

npm warn     @react-native/virtualized-lists@"0.79.2" from react-native@0.79.2

npm warn     node_modules/react-native

npm error code ERESOLVE

npm error ERESOLVE could not resolve

npm error

npm error While resolving: react-native@0.79.2

npm error Found: @types/react@18.3.23

npm error node_modules/@types/react

npm error   dev @types/react@"~18.3.12" from the root project

npm error

npm error Could not resolve dependency:

npm error peerOptional @types/react@"^19.0.0" from react-native@0.79.2

npm error node_modules/react-native

npm error   react-native@"0.79.2" from the root project

npm error   peer react-native@"*" from @react-native-picker/picker@2.11.0

npm error   node_modules/@react-native-picker/picker

npm error     @react-native-picker/picker@"2.11.0" from the root project

npm error   20 more (@react-native/virtualized-lists, ...)

npm error

npm error Conflicting peer dependency: @types/react@19.1.12

npm error node_modules/@types/react

npm error   peerOptional @types/react@"^19.0.0" from react-native@0.79.2

npm error   node_modules/react-native

npm error     react-native@"0.79.2" from the root project

npm error     peer react-native@"*" from @react-native-picker/picker@2.11.0

npm error     node_modules/@react-native-picker/picker

npm error       @react-native-picker/picker@"2.11.0" from the root project

npm error     20 more (@react-native/virtualized-lists, ...)

npm error

npm error Fix the upstream dependency conflict, or retry

npm error this command with --force or --legacy-peer-deps

npm error to accept an incorrect (and potentially broken) dependency resolution.

npm error

npm error

npm error For a full report see:

npm error /home/expo/.npm/_logs/2025-09-04T19_19_59_687Z-eresolve-report.txt

npm error A complete log of this run can be found in: /home/expo/.npm/_logs/2025-09-04T19_19_59_687Z-debug-0.log

npm ci exited with non-zero code: 1
```

--- a/app.json
+++ b/app.json
@@ -7,6 +7,7 @@
     "icon": "./assets/logo.png",
     "userInterfaceStyle": "light",
     "splash": {
+      "image": "./assets/logo.png",
       "resizeMode": "contain",
       "backgroundColor": "#ffffff"
     },

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
-    "@types/react": "~18.3.12",
+    "@types/react": "^19.1.12",
     "jsdoc": "^4.0.4",
     "standard-version": "^9.5.0",
     "typescript": "^5.1.3"

